homeassistant/components/portainer/strings.json

```
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "url": "[%key:common::config_flow::data::url%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        },
+        "data_description": {
+          "url": "The URL of your Portainer instance. For example: https://127.0.0.1:9000",
+          "port": "The port number of your Portainer instance",
+          "api_key": "The API key for authenticating with Portainer",
+          "verify_ssl": "Verify the SSL certificate of the Portainer instance"
+        },
+        "description": "You can retrieve your `API Key ` from the Portainer UI. Go to `My Account` > `Access Tokens` and create a new API key"
```

in the Portainer UI it is named access token, not API key, also it can't be retrieved, the token is only displayed once during creation but exising ones can't be retrieved.
```suggestion
        "description": "You can create an `access token` in the Portainer UI. Go to **My account > Access tokens** and click on **Add access token**"
```